agentMET4FOF/agents.py

```diff
@@ -8,23 +8,20 @@
 import time
 from collections import deque
 from io import BytesIO
-from multiprocessing.context import Process
-from threading import Thread, Timer
-from typing import Union, Dict, Optional
+from threading import Timer
+from typing import Dict, Optional, Union
+
 import matplotlib.figure
 import matplotlib.pyplot as plt
 import mpld3
 import networkx as nx
 import numpy as np
 import pandas as pd
-from mesa import Agent as MesaAgent
-from mesa import Model
+from mesa import Agent as MesaAgent, Model
 from mesa.time import BaseScheduler
-from osbrain import Agent as osBrainAgent
-from osbrain import NSProxy
-from osbrain import run_agent
-from osbrain import run_nameserver
+from osbrain import Agent as osBrainAgent, NSProxy, run_agent, run_nameserver
 from plotly import tools as tls
+
```

Comment:
This just drops unneeded `from multiprocessing.context import Process` and `from threading import Timer` and aligns the imports with PEP8 separating built-ins from third-party libraries from own modules.